### PR TITLE
shell completion for BASH

### PR DESCRIPTION
**Pull request by carstene1ns**
- Options expecting arbitrary arguments (invoke parsing lcf) are not completed of course.
- `--load-game-id` and `--start-map-id` do not complete currently, because they do not respect the `--project-dir` and `--save-dir` settings yet. Therefore they are useless for out of folder invocation and disabled.
- CMake integration was not worth for now, as this is being refactored currently.

### CODE REVIEW

**Comment by carstene1ns** [comment]
if you want to play around with it:
```bash
wget https://raw.githubusercontent.com/carstene1ns/easyrpg-player/shell_completion/resources/unix/bash-completion/easyrpg-player -O player-completion.sh
source player-completion.sh
easyrpg-player <TAB><TAB>
```

**Comment by Ghabry** [comment]
[gabriel@Schleppi tmp]$ source player-completion.sh 
bash: player-completion.sh: línea 44: error sintáctico cerca del elemento inesperado `('
bash: player-completion.sh: línea 44: `    --@(project-path|save-path))'

**Comment by carstene1ns** [comment]
Y U NO speak english with me?

File: resources/unix/bash-completion/easyrpg-player [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      #_filedir '@(lsd|esd)'
+      return
+      ;;
+    # set game directory
+    --@(project-path|save-path))
+      _filedir -d
+      return
+      ;;
+    # argument required but no completions available
+    --@(battletest|encoding|seed|start-position|start-party)| \
```
Comment (Ghabry) [inline resources/unix/bash-completion/easyrpg-player]: typo, must be battle-test and it appears twice

**Comment by Ghabry** [comment]
lgtm now. But has lots of bash and m4, so CC @fdelapena  ;)

**Comment by fdelapena** [comment]
> lgtm now. But has lots of bash and m4, so CC @fdelapena ;)

ditto :+1:

**Pull request merged**